tensorrt_llm/_torch/modules/fused_moe/quantization.py

```diff
@@ -1091,6 +1099,260 @@ def load_expert_w2_weight_scale_nvfp4(self, module: torch.nn.Module,
                     self.block_scales_dtype).reshape(orig_shape))
 
 
+class W4A8MXFP4FP8FusedMoEMethod(FusedMoEMethodBase):
+    """
+    Base class for W4A8MXFP4FP8 fused MoE methods for all backends.
+    """
+
+    def create_weights(self, module: torch.nn.Module, weight_dtype,
+                       weight_vec_size, block_scales_dtype,
+                       block_scales_vec_size):
+
+        module.scaling_vector_size = 32
+        # Divide by 32 because we use int64 to pack 16 fp4 values
+        w3_w1_weight_shape = (module.expert_size_per_partition,
+                              module.intermediate_size_per_partition * 2,
+                              module.hidden_size // weight_vec_size)
+        w2_weight_shape = (module.expert_size_per_partition, module.hidden_size,
+                           module.intermediate_size_per_partition //
+                           weight_vec_size)
+
+        # Divide by 4 because we use int32 to pack 4 fp8 values
+        # column parallel
+        w3_w1_weight_scale = nn.Parameter(
+            torch.ones(module.expert_size_per_partition,
```

Comment:
Is there a particular reason we use "ones" instead of empty here?

Reply:
I think empty is good. Just reuse those codes from NVFP4.

Reply:
Right, I am just wondering if we have a dummy weights mode that it might be relevant for. Otherwise, yes please update